examples/create-account-with-thresholdkey.js

```diff
@@ -0,0 +1,169 @@
+import {
+    Client,
+    PrivateKey,
+    AccountId,
+    Hbar,
+    AccountCreateTransaction,
+    TransferTransaction,
+    AccountBalanceQuery,
+    AccountDeleteTransaction,
+    KeyList,
+} from "@hashgraph/sdk";
+
+import dotenv from "dotenv";
+
+dotenv.config();
+
+// Removed pino logger initialization
+const logger = console;
+
+/**
+ * Step 0: Set up client connection to Hedera network
+ */
+
+async function main() {
+    if (
+        !process.env.OPERATOR_ID ||
+        !process.env.OPERATOR_KEY ||
+        !process.env.HEDERA_NETWORK
+    ) {
+        logger.error(
+            "Environment variables OPERATOR_ID, OPERATOR_KEY, and HEDERA_NETWORK are required.",
+        );
+        throw new Error("Missing required environment variables.");
+    }
+
+    const operatorId = AccountId.fromString(process.env.OPERATOR_ID);
+    const operatorKey = PrivateKey.fromStringED25519(process.env.OPERATOR_KEY);
+
+    let client;
+
+    // Adjust this as needed for your local network
+    const localNetworkConfig = {
+        "127.0.0.1:50211": new AccountId(3),
+    }
+
+    switch (process.env.HEDERA_NETWORK) {
+        case "mainnet":
+            client = Client.forMainnet();
+            break;
+        case "testnet":
+            client = Client.forTestnet();
+            break;
+        case "previewnet":
+            client = Client.forPreviewnet();
+            break;
+        case "local-node":
+            client = Client.forNetwork(localNetworkConfig);
+            break;
+        default:
+            logger.error("Unsupported HEDERA_NETWORK value.");
+            throw new Error(
+                "Unsupported HEDERA_NETWORK value. Set to 'mainnet', 'testnet', 'previewnet', or 'local-node'.",
+            );
+    }
```

Comment:
This is working good, but it involves quite a bit of code that doesn't directly relate to the focus of the test. Could you please refer to the other examples to see how to set the client in one line using the environment variable?